### PR TITLE
Fix PostgreSQL integration test error case tbl

### LINKED ISSUE #14840: PostgreSQL integration test error case tbl and empty_rules
## Bug Report

**For English only**, other languages will not accept.

Before report a bug, make sure you have:

- Searched open and closed [GitHub issues](https://github.com/apache/shardingsphere/issues).
- Read documentation: [ShardingSphere Doc](https://shardingsphere.apache.org/document/current/en/overview).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot reproduce it on current information, we will **close it**.

Please answer these questions before submitting your issue. Thanks!

### Which version of ShardingSphere did you use?

master

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?

ShardingSphere-Proxy

### Expected behavior

- PostgreSQL tbl

```bash
./mvnw -B install -f shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/pom.xml -Dit.adapters=proxy -Dit.databases=PostgreSQL -Dit.scenarios=tbl -Dit.env.type=DOCKER
```

- PostgreSQL empty_rules

```bash
./mvnw -B install -f shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/pom.xml -Dit.adapters=proxy -Dit.databases=PostgreSQL -Dit.scenarios=empty_rules -Dit.env.type=DOCKER
```

### Actual behavior

- PostgreSQL tbl

```bash
[ERROR] Failures: 
[ERROR]   GeneralDQLIT.assertExecuteQuery:66->assertExecuteQueryForStatement:77->BaseDQLIT.assertResultSet:67->BaseDQLIT.assertRows:95->BaseDQLIT.assertRow:108->BaseDQLIT.assertObjectValue:124 
Expected: is "100000"
     but: was "100001"
[ERROR]   GeneralDQLIT.assertExecuteQuery:68->assertExecuteQueryForPreparedStatement:87->BaseDQLIT.assertResultSet:67->BaseDQLIT.assertRows:95->BaseDQLIT.assertRow:108->BaseDQLIT.assertObjectValue:124 
Expected: is "100000"
     but: was "100001"
[ERROR]   GeneralDQLIT.assertExecute:96->assertExecuteForStatement:107->BaseDQLIT.assertResultSet:67->BaseDQLIT.assertRows:95->BaseDQLIT.assertRow:108->BaseDQLIT.assertObjectValue:124 
Expected: is "100000"
     but: was "100001"
[ERROR]   GeneralDQLIT.assertExecute:98->assertExecuteForPreparedStatement:119->BaseDQLIT.assertResultSet:67->BaseDQLIT.assertRows:95->BaseDQLIT.assertRow:108->BaseDQLIT.assertObjectValue:124 
Expected: is "100000"
     but: was "100001"
[ERROR] Errors: 
[ERROR]   GeneralRQLIT.<init>:47->BaseRQLIT.<init>:40->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   GeneralRQLIT.<init>:47->BaseRQLIT.<init>:40->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   GeneralRQLIT.<init>:47->BaseRQLIT.<init>:40->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   GeneralRQLIT.<init>:47->BaseRQLIT.<init>:40->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   GeneralRQLIT.<init>:47->BaseRQLIT.<init>:40->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   GeneralRQLIT.<init>:47->BaseRQLIT.<init>:40->SingleITCase.<init>:52 » IllegalArgument
[INFO] 
[ERROR] Tests run: 528, Failures: 4, Errors: 6, Skipped: 0
```

- PostgreSQL empty_rules

```bash
[ERROR] Errors: 
[ERROR]   GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[ERROR] org.apache.shardingsphere.test.integration.engine.it.rdl.GeneralRDLIT.assertExecute[proxy: empty_rules -> PostgreSQL -> Literal -> SHOW ENCRYPT TABLE RULE t_user](org.apache.shardingsphere.test.integration.engine.it.rdl.GeneralRDLIT)
[ERROR]   Run 1: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   Run 2: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[INFO] 
[ERROR] org.apache.shardingsphere.test.integration.engine.it.rdl.GeneralRDLIT.assertExecute[proxy: empty_rules -> PostgreSQL -> Literal -> SHOW READWRITE_SPLITTING RULES](org.apache.shardingsphere.test.integration.engine.it.rdl.GeneralRDLIT)
[ERROR]   Run 1: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   Run 2: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   Run 3: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[INFO] 
[ERROR] org.apache.shardingsphere.test.integration.engine.it.rdl.GeneralRDLIT.assertExecute[proxy: empty_rules -> PostgreSQL -> Literal -> SHOW SCHEMA RESOURCES](org.apache.shardingsphere.test.integration.engine.it.rdl.GeneralRDLIT)
[ERROR]   Run 1: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   Run 2: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   Run 3: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[INFO] 
[ERROR] org.apache.shardingsphere.test.integration.engine.it.rdl.GeneralRDLIT.assertExecute[proxy: empty_rules -> PostgreSQL -> Literal -> SHOW SHARDING BINDING TABLE RULES](org.apache.shardingsphere.test.integration.engine.it.rdl.GeneralRDLIT)
[ERROR]   Run 1: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   Run 2: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   Run 3: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[INFO] 
[ERROR] org.apache.shardingsphere.test.integration.engine.it.rdl.GeneralRDLIT.assertExecute[proxy: empty_rules -> PostgreSQL -> Literal -> SHOW SHARDING BROADCAST TABLE RULES](org.apache.shardingsphere.test.integration.engine.it.rdl.GeneralRDLIT)
[ERROR]   Run 1: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   Run 2: GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[INFO] 
[ERROR]   GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
[ERROR]   GeneralRDLIT.<init>:49->BaseRDLIT.<init>:43->SingleITCase.<init>:52 » IllegalArgument
```

### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.

- set environment

```bash
./mvnw -B clean install -am -pl shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite -Pit.env.docker -DskipTests
```

- PostgreSQL tbl

```bash
./mvnw -B install -f shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/pom.xml -Dit.adapters=proxy -Dit.databases=PostgreSQL -Dit.scenarios=tbl -Dit.env.type=DOCKER
```

- PostgreSQL empty_rules

```bash
./mvnw -B install -f shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/pom.xml -Dit.adapters=proxy -Dit.databases=PostgreSQL -Dit.scenarios=empty_rules -Dit.env.type=DOCKER
```

### PR DESCRIPTION
**Pull request by soulasuna**
## Fix PostgreSQL integration test error case tbl

Fix #14840

Changes proposed in this pull request:
- Fix PostgreSQL integration test error case tbl

### CODE REVIEW

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dql/dql-integration-test-cases.xml [hunk 1]
```diff
@@ -602,7 +602,7 @@
         <assertion parameters="1:int" expected-data-file="select_sql_by_id_across_single_tables_with_encrypt_user_id.xml" />
     </test-case>
 
-    <test-case sql="SELECT o.item_id, o.order_id, o.creation_date, s.id FROM t_order_item o INNER JOIN t_single_table s ON o.order_id = s.id" scenario-types="db,tbl">
+    <test-case sql="SELECT o.item_id, o.order_id, o.creation_date, s.id FROM t_order_item o INNER JOIN t_single_table s ON o.order_id = s.id Order by o.item_id " scenario-types="db,tbl">
```
Comment (strongduanmu) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dql/dql-integration-test-cases.xml:605]: @soulasuna Use `ORDER BY` may be better.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dql/dql-integration-test-cases.xml (hunk 1 above)
Comment (soulasuna) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/resources/cases/dql/dql-integration-test-cases.xml:605]: I will fix it

**Comment by RaigorJiang** [comment]
The problem of CI failure has been fixed in #14847, please merge the master branch and push again, thank you!

**Comment by soulasuna** [comment]
> The problem of CI failure has been fixed in #14847, please merge the master branch and push again, thank you!

Thanks for your remind.

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**